src/components/base-address/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@bcrs-shared-components/base-address",
+  "version": "2.0.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@bcrs-shared-components/mixins": "^1.1.16",
+    "lodash.uniqueid": "^4.0.1",
+    "vue-property-decorator": "^9.1.2",
+    "vuelidate": "^0.7.4",
+    "vue-plugin-helper-decorator": "^0.0.11"
+  }
```

Comment:
Do you depend on Vue?

Also please move appropriate packages to "devDependencies".

Reply:
If I had to guess, I would think it depends on Vue, but not entirely sure. Seems to work fine without adding vue to the dependencies, but I can add it to be safe?

Reply:
Yes please. Can't hurt (I think).